src/ompl/geometric/planners/bitstar/datastructures/src/IntegratedQueue.cpp

```diff
@@ -1578,7 +1578,7 @@ namespace ompl
 
         bool BITstar::IntegratedQueue::isCostBetterThan(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
-            return a.value() < b.value();
+            return opt_->isCostBetterThan(a, b);
```

Comment:
This is the reason why I defined a `IntegratedQueue::isCostBetterThan` function, so if someone needed to use the `OptimizationObjective` version (or just change the ordering) there was only one place to do it. Also why everything else is defined in terms of this function. :smile:

No, unfortunately I currently have more code duplication between `BITstar` and `IntegratedQueue`, but that's really a temporary situation until I get a chance to propose a better heuristic class than what's currently inside `OptimizationObjective`.